vaadin-spring/src/main/java/com/vaadin/flow/spring/security/RequestUtil.java

```diff
@@ -109,6 +109,11 @@ public boolean isAnonymousEndpoint(HttpServletRequest request) {
     public boolean isAnonymousRoute(HttpServletRequest request) {
         String vaadinMapping = configurationProperties.getUrlMapping();
         String requestedPath = getRequestPathInsideContext(request);
+        if (requestedPath.startsWith("/")) {
```

Comment:
What is this public API that has no javadocs ? Please fix that too.

Reply:
Fixed